postman-test-collections/fn-pushnotif-test.postman_collection.json

```diff
@@ -1,352 +1,352 @@
-							"});",
-							""
-						],
-						"type": "text/javascript"
-					}
-				}
-			],
-			"request": {
-				"method": "POST",
-				"header": [
-					{
-						"key": "Content",
-						"value": "application/json",
-						"type": "text"
-					}
-				],
-				"body": {
-					"mode": "raw",
-					"raw": "{\n    \"input\": {{notify_msg_json}}\n}",
-					"options": {
-						"raw": {
-							"language": "json"
-						}
-					}
-				},
-				"url": "{{basePath}}/admin/functions/HandleNHNotificationCall"
-			},
-			"response": []
-		},
-		{
-			"name": "Can delete installation",
-			"event": [
-				{
-					"listen": "test",
-					"script": {
-						"exec": [
-							"pm.test(\"Status code is 202\", function () {",
-							"    pm.response.to.have.status(202);",
-							"});",
-							"",
-							"",
-							"",
-							""
-						],
-						"type": "text/javascript"
-					}
-				},
-				{
-					"listen": "prerequest",
-					"script": {
-						"exec": [
-							"var instId = pm.variables.get(\"installationId\");",
-							"",
-							"var delete_msg = {",
-							"            kind: \"DeleteInstallation\",",
-							"            installationId: instId,",
-							"        };",
-							"",
-							"console.log(delete_msg);",
-							"",
-							"var delete_msg_json = JSON.stringify(JSON.stringify(delete_msg));",
-							"",
-							"pm.collectionVariables.set(\"delete_msg_json\", delete_msg_json);"
-						],
-						"type": "text/javascript"
-					}
-				}
-			],
-			"request": {
-				"method": "POST",
-				"header": [
-					{
-						"key": "Content",
-						"value": "application/json",
-						"type": "text"
-					}
-				],
-				"body": {
-					"mode": "raw",
-					"raw": "{\n    \"input\": {{delete_msg_json}}\n}",
-					"options": {
-						"raw": {
-							"language": "json"
-						}
-					}
-				},
-				"url": "{{basePath}}/admin/functions/HandleNHNotificationCall"
-			},
-			"response": []
-		}
-	],
-	"event": [
-		{
-			"listen": "prerequest",
-			"script": {
-				"type": "text/javascript",
-				"exec": [
-					""
-				]
-			}
-		},
-		{
-			"listen": "test",
-			"script": {
-				"type": "text/javascript",
-				"exec": [
-					""
-				]
-			}
-		}
-	],
-	"variable": [
-		{
-			"key": "basePath",
-			"value": "http://localhost:7007"
-		},
-		{
-			"key": "installationId",
-			"value": "cee277cd50dff7cfa143a6a7fa4f89c4f4095dc4614b1d249b9bd9c8070a91dc"
-		},
-		{
-			"key": "pushChannel",
-			"value": "epbR4dxGSuGteiWkAa5tiS:APA91bHhXQfnOAHudsXmYG9xzqNonrBylHDEzwFAFRoradEof15sRp-U2A-nlAnEJaq_oi44TDV6tGCVzTYzYsNN3EIxREWEnFJtT4uI-AiGK0KPe8T1LKPhOBrhHwwr3SWl-RF3rPjZ"
-		},
-		{
-			"key": "platform",
-			"value": "gcm"
-		},
-		{
-			"key": "create_update_msg_json",
-			"value": ""
-		},
-		{
-			"key": "notify_msg_json",
-			"value": ""
-		},
-		{
-			"key": "delete_msg_json",
-			"value": ""
-		},
-		{
-			"key": "msgNumber",
-			"value": ""
-		}
-	]
-}
\ No newline at end of file
+  "info": {
```

Comment:
This file has only tab fixing and i replaced `MyServiceId` with `base` whenever it was present. I guess you can skip this file :)